### DISCUSSION TITLE
Dont work. ua/ru

### DISCUSSION THREAD

**Discussion by yarosh69** [ORelio/Minecraft-Console-Client#2365] [Q&A] [answered]
How fix that? #
![ScreenShot_20221209012338](https://user-images.githubusercontent.com/120141684/206580380-f495c308-388c-4089-8e52-6d9fae2b715b.png)

**Comment by milutinke** [comment, marked as answer]
Use Alacritty terminal in order to have colors.
As for the version, you can manually specify a version of the server in the `MinecraftClient.ini`, take a look at this https://mccteam.github.io/guide/configuration.html#main-advanced-section
